conjure-go-verifier/ignored-test-cases.yml

```diff
@@ -111,6 +111,9 @@ client:
     - "null"
     receiveSetDoubleAliasExample:
     - "[100, 10.0, \"NaN\", \"Infinity\", \"-Infinity\"]"
+    receiveEnumExample:
```

Comment:
Note that these two cases are similarly ignored by conjure-java as well: https://github.com/palantir/conjure-java/blob/c9de473727fe1e685c3ad7e24f3c2023c3e7f009/conjure-java-client-verifier/src/test/resources/ignored-test-cases.yml#L107-L109.